test/cmdlineTests/linking_strict_assembly/args

```diff
@@ -1 +1 @@
---strict-assembly --libraries contract/test.sol:L=0x1234567890123456789012345678901234567890
+--strict-assembly --bin --asm --ewasm --ewasm-ir --ir-optimized --libraries contract/test.sol:L=0x1234567890123456789012345678901234567890
```

Comment:
In case of all the linker tests we can reduce this to `--bin`. All other outputs are not affected.
```suggestion
--strict-assembly --bin --libraries contract/test.sol:L=0x1234567890123456789012345678901234567890
```

Reply:
Alright, but this also means I have to change the expected output in the tests.